### ISSUE TITLE
Slight audio crackle/stutter/glitching during fades/transitions in Pokémon Mystery Dungeon: Blue Rescue Team

### ISSUE THREAD

**Issue by fighuass** [Arisotura/melonDS#791]
OS: Kubuntu 20.04
Version: 0.9

Problem: The game itself works perfectly fine, but whenever the game fades to black (like during transitions in cutscenes, changing floors, moving to a different area, etc) the music seems to glitch out somewhat (it sounds like crackling or stuttering). Beyond that there's no problems with the audio.

**Comment by RSDuck** [comment]
The problem really seems to be just that melonDS can't keep up when the emulated processor is 100% utilised (like in those segments where new data is loaded).

There's another layer to this, ARM9 timing is pretty bad (see e.g. https://github.com/Arisotura/melonDS/issues/776) and we tend to run the processor faster than it actually is, so the game performs more work per emulated frame than it could do on a real DS.

Have you enabled the JIT recompiler? It should give a speed boost which should atleast mitigate this.

**Comment by fighuass** [comment]
Yes, haven't noticed any difference between having it turned on or off, though.

**Comment by iamthelaw2041** [comment]
@fighuass Is this still an issue for you? I don't get any crackling at all. 

![Capture](https://user-images.githubusercontent.com/94656803/144727703-fb480769-40d6-496f-965d-dbede2c43c8e.PNG)
